src/adapters/mpd.cpp

```diff
@@ -476,4 +483,6 @@ namespace mpd {
   // }}}
 }
 
+#undef LOG_BOOL
```

Comment:
Is this supposed to be `TRACE_BOOL`?

Reply:
Yes, it is. Must have missed that when I renamed the rest.